libs/application-generic/src/usecases/select-variant/select-variant.spec.ts

```diff
@@ -28,8 +29,9 @@ describe('select variant', function () {
     selectVariantUsecase = new SelectVariant(
       // @ts-ignore
       new ConditionsFilter(),
-      new MessageTemplateRepository()
+      new MessageTemplateRepository(),
       // @ts-ignore
+      new NormalizeVariables()
```

Comment:
The `@ts-ignore` here seems to have been applied to nothing before this diff. Can we remove it?

```suggestion
      new NormalizeVariables()
```